src/frequenz/sdk/actor/power_distributing/_component_managers/_pv_inverter_manager/_pv_inverter_manager.py

```diff
@@ -137,6 +145,10 @@ async def distribute_power(self, request: Request) -> None:
             reverse=True,
         )
 
+        if len(working_components) == 0:
+            _logger.error("No inverters available for power distribution. Aborting.")
+            return
+
         num_components = len(working_components)
```

Comment:
you might want to combine them
```suggestion
        num_components = len(working_components)
        if num_components == 0:
            _logger.error("No inverters available for power distribution. Aborting.")
            return
```